R/utils-custom.R

```diff
@@ -12,22 +12,33 @@
 theme_tiltplot <- function() {
   theme_classic() +
     theme(
-      plot.title = element_text(hjust = 0.5, size = 16),
-      axis.title = element_text(size = 12),
-      axis.text = element_text(size = 10),
-      legend.title = element_text(size = 12),
-      legend.text = element_text(size = 10),
+      plot.title = element_text(hjust = 0.5, size = 16, family = tilt_headline_font()),
+      strip.text = element_text(family = tilt_text_font()),
+      axis.title = element_text(size = 10, family = tilt_text_font()),
+      axis.text = element_text(size = 10, family = tilt_text_font()),
+      legend.title = element_text(size = 12, family = tilt_text_font()),
+      legend.text = element_text(size = 10, family = tilt_text_font()),
       legend.position = "bottom",
       panel.grid.major.y = element_blank(),
       panel.grid.minor.y = element_blank(),
       panel.spacing.y = unit(0.5, "lines")
     )
 }
 
-score_colors <- function(...) c("low" = "#007F00", "medium" = "#FFC300", "high" = "#FF5733")
+tilt_text_font <- function() "Roboto"
+
+tilt_headline_font <- function() "Roboto Condensed"
```

Comment:
Some checks fail with a message indicating a missing font.

<details/>

```
Error: Running examples in ‘tiltPlot-Ex.R’ failed
The error most likely occurred in:
> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: bar_plot_emission_profile
> ### Title: Create a horizontal stacked bar chart without financial data
> ### Aliases: bar_plot_emission_profile
> 
> ### ** Examples
> 
> benchmarks <- c("all", "unit", "isic_4digit")
> bar_plot_emission_profile(without_financial, benchmarks)
Warning in grid.Call(C_stringMetric, as.graphicsAnnot(x$label)) :
  font family 'Roboto' not found in PostScript font database
Warning in grid.Call(C_stringMetric, as.graphicsAnnot(x$label)) :
  font family 'Roboto' not found in PostScript font database
```

</details>

That's a rabbit hole you may need to explore, but for now you might be able to avoid it by adding a fall back to an a common font with something like `ifelse(exists(rare_font), rare_font, common_font)`.

This is inspired by the main example I see in the ragg package

https://ragg.r-lib.org/

![image](https://github.com/2DegreesInvesting/tiltPlot/assets/5856545/0530e795-9c0d-46a3-b097-4a91194e7ee7)